sx-tab.el

```diff
@@ -232,6 +232,24 @@ If SITE is nil, use `sx-default-site'."
   nil t)
 
 
+;;; Starred
+(sx-tab--define "Starred"
+  (lambda (page)
+    (sx-method-call 'me
+      :page page
+      :site sx-question-list--site
+      :auth t
+      :submethod 'favorites
+      :filter sx-browse-filter)))
+;;;###autoload
+(autoload 'sx-tab-featured
+  (expand-file-name
+   "sx-tab"
+   (when load-file-name
+     (file-name-directory load-file-name)))
+  nil t)
```

Comment:
Now-a-days `make-autoloads` fully expands macros it runs into, and it recursively autoloads the contents of a `progn`. So we could just `;;;###autoload` the calls to `sx-tab--define`.

The way it was before, `progn` wasn't understood, so there's not much we can do I think. Maybe it still expands macros, might be worth checking.